website/src/plugins/github-metadata-plugin.ts

```diff
@@ -0,0 +1,39 @@
+/**********************************************************************
+ * Copyright (C) 2025 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ ***********************************************************************/
+
+import type { Plugin } from '@docusaurus/types';
+
+import type { GitHubMetadata } from './github-metadata';
+import { GitHubService } from './github-service';
+
+export default async function githubMetadataPlugin(): Promise<Plugin<GitHubMetadata>> {
+  const githubService = new GitHubService('podman-desktop', 'podman-desktop');
```

Comment:
IMHO it could reuse the stuff that is defined for the project there

https://github.com/podman-desktop/podman-desktop/blob/main/package.json#L4

as this is where we declare the repository of Podman Desktop

Reply:
Interesting but Octokit requires owner and repo to be parsed so I don't know if we can reuse without a hack?

Reply:
<img width="950" height="709" alt="Screenshot 2025-07-16 at 16 09 57" src="https://github.com/user-attachments/assets/6c28eb36-4dc8-42bf-bbfb-f0f8d32b0ea5" />

Reply:
import rootPackage from '../..I don't know how many ../../../package.json' with { type: 'json' };

then fromUrl(rootPackage.repository)